example/android/settings.gradle

```diff
@@ -1,11 +1,36 @@
+
+
+pluginManagement {
+    def flutterSdkPath = {
+        def properties = new Properties()

```

Comment:
this too ?
What version did you set your to?

Reply:
Hi @bensonarafat,

Thank you for reviewing my pull request.

I made those changes because I'm using Flutter version 3.24, which required updating the Gradle configuration to run the project successfully. Specifically, these changes were necessary due to the deprecated imperative application of Flutter's Gradle plugins. You can find more details in this Flutter article: [Deprecated imperative apply of Flutter's Gradle plugins](https://docs.flutter.dev/release/breaking-changes/flutter-gradle-plugin-apply).

As I'm new to open-source contributions, I appreciate any guidance on best practices. If maintaining the same Gradle version as the original project is preferred, I’m happy to revert the changes or adjust them accordingly.

Thanks again for your feedback and patience!

Reply:
It won't be best to increase the flutter version. 
Can we keep it as is for now? Others are using the package. 

We can increase it later, but now is not the best time @RotRot-pi